lib/features/scanner/telos_signing_manager.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:dart_esr/dart_esr.dart';
+
+class SeedsESR {
+
+  SigningRequestManager manager;
+
+  List<Action> actions;
+
+  SeedsESR({String uri}) {
+    manager = TelosSigningManager.from(uri);
+  }
+
+  Future<void> resolve({String account}) async {
+    this.actions = await manager.fetchActions(account: account);
+  }
+
+  Action firstAction() {
+    return actions[0];
+  }
+}
+
+extension TelosSigningManager on SigningRequestManager {
+    static SigningRequestManager from(String uri) {
+      return SigningRequestManager.from(uri,
+      options:
+          defaultSigningRequestEncodingOptions(nodeUrl: 'https://api.eos.miami'));
```

Comment:
I think i have seen this string `https://api.eos.miami` in  multiple places now. 
Should we move this to a shared file

Reply:
lol - oversight, should use our preferred node from the service, although eos.miami rocks

Reply:
@gguijarro-c-chwy Hmm... I actually don't know how to get the node endpoint into this class extension... this might then have to become a service or IDK